contracts/CO30_Felix-Emerson_SameDayDelivery.zargabad/edit_me/briefing/mission.html

```diff
@@ -0,0 +1,9 @@
+Contractors have been hired by Vrana Corpration to assist in the transport of food and medical supplies to designated IDAP camps in order to assist the locals.
+
+The situation in greater Takistan is tense, and many locals have begun to arm themselves with surplus equipment to defend themselves and their homes after recent pullouts. IDAP has entered the scene in an attempt to help out those in need by providing food and medical supplies.
```

Comment:
This should really be at the top of situation.html imo, it's providing situation context, not mission details